docs/ecology.md

```diff
@@ -1 +1,176 @@
-# Victory
\ No newline at end of file
+# Victory
+
+Victory is a collection of composable React components you can use to build interactive data visualizations.
+
+### Overarching benefits and motivations
+
+1. Bridge the gap between beginners (more likely to use Chart.js or Highcharts) and experts (more likely to use D3)
+  * Provide a gradient of customization, from none (just pass in data) to full (build your own visualization features)
+  * Provide interactive documentation to help developers get visualizations into their projects (see [ecology.js](https://github.com/FormidableLabs/ecology)) see [victory-chart](projects.formidablelabs.com/victory-chart/) for an example
+  * Provide better developer UX generally, for both instantiating data visualizations and creating them
+2. Leverage the power of React to handle DOM, component lifecycle, state, computed styles, etc.
+3. Make data visualizations first class citizens in the open source community
```

Comment:
I would vote to nuke this list. maybe this goes over better in person, but I dont think a list of our tooling belongs on the lander. We can call out builder in the readme